web/frontend/src/components/gantry/index.svelte

```diff
@@ -50,6 +64,50 @@ const increment = (axis: number, amount: number) => {
   $robotClient.gantryService.moveToPosition(req, displayError);
 };
 
+const gantryModifyAllDoMoveToPosition = () => {
+  const gantry = status!;
+  const newList : number[] = Array.from({ length: gantry.positions_mm.values.length });
+  const newPieces = modifyAllStatus.pieces;
+
+  for (const [i, newPiece] of newPieces.entries()) {
+    newList[i] = newPiece!.pos;
+  }
```

Comment:
This could probably be simplified to:

```
// Pieces is probably more descriptive than newList
const pieces = modifyAllStatus.pieces.map((piece) => piece.pos)
```

`.map()` will create a new array copy just like `Array.from` but we're compressing this all into one line